show/muxcable.py

```diff
@@ -2187,9 +2187,8 @@ def get_grpc_cached_version_mux_direction_per_port(db, port):
 
     trans_dir = {}
     trans_dir = trans_info_full_dict[asic_index]
```

Comment:
shall  we add a get call here, that would be easier to read the code below
```
trans_dir = trans_info_full_dict.get(asic_index, None)
```

Reply:
@vdahiya12 Makes sense. Added it now.